packages/components/app/styles/components/pagination.scss

```diff
@@ -10,19 +10,43 @@
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
 }
 
+// Responsive layout
```

Comment:
This is just a simple stacked layout. We can change it from this if we want some other layout instead.

Reply:
@KristinLBradley @cveigt what do you think of this approach/layout? It would optimize even more the vertical space (precious in small screens context) and it's also very simple to achieve:

<img width="314" alt="screenshot_2282" src="https://user-images.githubusercontent.com/686239/213406245-d31c4322-b2c2-44a8-9cbe-005cb44d2ec8.png">

Reply:
I will try it, thanks

Reply:
I pushed revised layout styles. Screen shots of both layout approaches are included in this PR's description. (Original, 3 row centered & stacked vs. updated, 2 row centered & stacked layouts)